api/krusty/sortordertransformer_test.go

```diff
@@ -0,0 +1,328 @@
+package krusty_test
```

Comment:
CI is now pointing out that this file is missing the license header. Please run `./hack/add-license.sh run` to add it. Hopefully this will be the last issue! 🤞